src/App.js

```diff
@@ -1,23 +1,21 @@
-import logo from './logo.svg';
 import './App.css';
+import Home from './pages/Home';
+import { Link, Route, Routes } from 'react-router-dom';
+import Products from './pages/Products';
+import Product from './pages/Product';
+//import { Navbar, NavbarHeader, NavbarItems, NavItem } from 'react-bootstrap-navbar';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Link to="/">Home</Link>
+      <Link to="/products">Products</Link>
+      <hr /> 
+      <Routes>
+          <Route path="/" element={<Home/>} />
+          <Route path="/products" element={<Products/>} />
+          <Route path="/products/:product_id" element={<Product/>} />
```

Comment:
This works, but it's better to use a nested route
```
<Route path="/products" element={<Products/>} >
          <Route path=":productId" element={<Product/>} />
</Route>
```
You'd want the product ID to be camelCase too